src/assets/styles/pages/_projects-tools.scss

```diff
@@ -15,11 +15,6 @@
   .project {
     margin-top: rem(48);
 
-    .thumbnail {
```

Comment:
Removing the control of `width` and `height` does fixed the distorted image at screen resizing. However, these values can go fairly large at resizing. In my test, the width goes over 500px and same as the height. @jhung probably should review this PR to make sure this is acceptable.

Another method to fix is to keep `width` and `height` values and add `object-fit: contain` to maintain its aspect ratio.